libkirk/kselftests.py

```diff
@@ -116,6 +116,43 @@ async def get_suites(self, sut: SUT) -> list:
 
         return ["cgroup", "bpf"]
 
+    async def find_command(self, sut: SUT, command: str) -> Test:
+        if not sut:
+            raise ValueError("SUT is None")
+
+        if not command:
+            raise ValueError("command is empty")
+
+        cmd_args = command.split(" ")
+        suite_folder = None
+
+        for suite in await self.get_suites(sut):
+            folder = os.path.join(self._root, suite)
+            binary = os.path.join(folder, cmd_args[0])
+
+            ret = await sut.run_command(f"test -f {binary}")
+            if ret["returncode"] == 0:
+                suite_folder = folder
+                break
+
+        cwd = None
+        env = None
```

Comment:
I hope this will not clear environment variables set outside of kirk before kirk is run.

Reply:
You mean environment variables set by `-e` ? Those are used only by the suites. Otherwise it's using the current env variables for the default shell `/bin/sh`

Reply:
No, I mean current environment variables set by user before running kirk.

Reply:
The commands are running with a `/bin/sh` default set of environment variables. The way we have to inject new environment variables in kirk is to use:
`--env ENV, -e ENV     List of key=value environment values separated by ':'`

The reason is that we don't want to inject any local environment variable in the remote SUT.
Maybe we can use `--env` to inject more environment variables and add that support also for commands. What do you think?